src/chapters/databases-part2/introduction.rst

```diff
@@ -0,0 +1,39 @@
+Introduction
+============
+
+Welcome to the end of LaunchCode's LCHS course! In this chapter, we will use
+our Python, web development, and database skills to build a game application.
+Each of the following pages includes a video tutorial, sample code, and a text
+summary for one piece of the project.
+
+.. index:: ! minimum viable product
+
+The goal is to build a bare-bones Minesweeper game. Consider this a prototype
+for a more refined program. By the end of the chapter, we will have a working
+game, but it will contain a few gaps and rough edges. Repeated playing of the
+game will identify improvements that *could* be made, but these won't prevent
+the app from operating.
+
+Once you have your prototype up and running, feel free to make changes and add
+different features to make the project your own!
+
+The video clip below demonstrates what the finished project will look like. It
+also provides game instructions for those who might not be familiar with how to
+play Minesweeper.
+
+.. raw:: html
+
+   <section class="vid_box">
+      <iframe class="vid" src="https://www.youtube.com/embed/n2tXf33yAQk" frameborder="1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </section>
+
+Project Setup
+-------------
+
+#. Launch Visual Studio Code and open your ``local_practice`` directory.
+#. Clone the `Minesweeper starter code <https://github.com/LaunchCodeEducation/LCHS_minesweeper>`__
+   from GitHub. The project folder is called ``LCHS_minesweeper``.
```

Comment:
So I originally downloaded the repo to my computer and had to create the gitignore file, not sure why?

I just tested cloning and it is there.  Not sure if we need to make a note or not.

Reply:
I just double checked the clone sequence, and the `.gitignore` file came with the download.  I think we can leave this one alone.

Even if the file doesn't clone to the kids computers, they will still be able to run the program.  Of course, their Flask, environment, and database files will all git recorded by the VCS.  At this point in the course, I think we can leave it to them to correct any glitches.